packages/share_plus/share_plus/example/integration_test/share_plus_test.dart

```diff
@@ -14,4 +14,9 @@ void main() {
   testWidgets('Can launch share', (WidgetTester tester) async {
     expect(Share.share('message', subject: 'title'), completes);
   });
+
+  testWidgets('Tests if the expected non-null result is returned',
```

Comment:
I mean, it is an improvement :) but could you add a reference to the method name shareWithResult?

e.g. "Can launch shareWithResult and result is not null."

Reply:
Yes!!!